packages/SidePanel/tests/SidePanel.spec.js

```diff
@@ -88,15 +87,96 @@ describe("SidePanel", () => {
     });
   });
 
+  describe("SidePanel.Group", () => {
+    it("should render with multiple sidepanels", () => {
+      const { getAllByTestId } = renderReactTestingLibrary(
+        <SidePanel.Group>
+          <SidePanel onClose={noop} isOpen>
+            <SidePanel.Header>With Header</SidePanel.Header>
+            <SidePanel.Overlay />
+          </SidePanel>
+          <SidePanel onClose={noop} isOpen>
+            <SidePanel.Header>With Header</SidePanel.Header>
+            <SidePanel.Overlay />
+          </SidePanel>
+          <SidePanel onClose={noop} isOpen>
+            <SidePanel.Header kind="primary">With Header</SidePanel.Header>
+            <SidePanel.Overlay />
+          </SidePanel>
+        </SidePanel.Group>
+      );
+      // data-pka-anchor
+      expect(getAllByTestId("heading")).toHaveLength(3);
```

Comment:
@slee1130 Use the `sidepanel` to determine number of sidepanels and not the heading.

Reply:
done